packages/create-nx-plugin/bin/detect-invoked-package-manager.ts

```diff
@@ -1,4 +1,6 @@
-export type PackageManager = 'yarn' | 'pnpm' | 'npm';
+const packageManagerList = ['npm', 'pnpm', 'yarn'] as const;
```

Comment:
The order of the list has changed. Can you please ensure the order is:
1. `pnpm`
2. `yarn`
3. `npm`